src/lightning/fabric/utilities/logger.py

```diff
@@ -129,10 +130,12 @@ def _sanitize_params(params: Dict[str, Any]) -> Dict[str, Any]:
 
     """
     for k in params:
-        # convert relevant np scalars to python types first (instead of str)
-        if isinstance(params[k], (np.bool_, np.integer, np.floating)):
-            params[k] = params[k].item()
-        elif type(params[k]) not in [bool, int, float, str, Tensor]:
+        if _NUMPY_AVAILABLE:
+            import numpy as np
```

Comment:
We forgot to do this in seed.py in the previous PR. Could you also edit utilities/seed.py to move the numpy import from the global import to local import like you did here?